### DISCUSSION TITLE
S3 API Compatible Object Storage Plans?

### DISCUSSION THREAD

**Discussion by PizzaConsole** [PiranhaCMS/piranha.core#1835] [Ideas]
Like the title says: Are there any plans to add S3 Compatible Object Storage to use instead of Azure blobs or Local storage?

**Comment by tidyui** [comment]
There is another user who has developed a provider for S3, not sure which version of Piranha it is updated to, but the code is available on GitHub. You can find the link here:

https://github.com/PiranhaCMS/piranha.modules

**Reply by PizzaConsole** [reply]
Oh awesome! I actually had already built my own! I suppose I can put it there if it helps someone else out.
